src/explorer/components/NodeFilter.vue

```diff
@@ -62,15 +64,9 @@ export default class InFilter extends Vue {
       key: this.filterKey,
       value,
     });
-
-    // reset to the first page
-    this.$store.commit(
-      "explorer/" + MutationTypes.SET_NODES_TABLE_PAGE_NUMBER,
-      1
-    );
-
-    // load nodes with the changes
-    this.$store.dispatch(ActionTypes.REQUEST_NODES);
+    if(!this.invalid){
```

Comment:
Can we make the condition at the top of the function? Or you Have a different opinion?

Reply:
sure.